ci/tests/general/general_alba.py

```diff
@@ -381,11 +398,14 @@ def initialise_disks(alba_backend, nr_of_disks, disk_type):
         assert len(uninitialized_disk_names) >= nr_of_disks_to_init, "Not enough disks to initialize!"
 
         disks_to_init = GeneralAlba.filter_disks(uninitialized_disk_names, nr_of_disks_to_init, disk_type)
-        assert len(disks_to_init) >= nr_of_disks_to_init, "Not enough disks to initialize!"
-
-        grid_ip = General.get_config().get('main', 'grid_ip')
-        alba_node = AlbaNodeList.get_albanode_by_ip(grid_ip)
-        failures = AlbaNodeController.initialize_disks(alba_node.guid, disks_to_init)
+        amount_disks_to_init = 0
+        for ip in disks_to_init:
```

Comment:
I'd expect to be able to initialize 1 disk on node1, 2 disks on node2, 3 disks on node3
Meaning, much more configurability